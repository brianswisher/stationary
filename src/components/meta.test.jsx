import React from "react";
import renderer from "react-test-renderer";

import Meta from "./meta";

const data = ["a", "b"];

describe("Tags", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Meta popularity={4} reliability={80} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
