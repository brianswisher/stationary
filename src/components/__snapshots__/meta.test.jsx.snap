// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tags renders correctly 1`] = `
<span
  className="meta"
>
  <span
    className="badge popular"
  >
    popular
  </span>
  <span
    className="badge reliable"
  >
    reliable
  </span>
</span>
`;
